docs/doxygen/Doxyfile

```diff
@@ -1117,8 +1117,8 @@ EXAMPLE_RECURSIVE      = NO
 # that contain images that are to be included in the documentation (see the
 # \image command).
 
-IMAGE_PATH             = ../../data/doxygenInputs \
-                         ../../data/doxygenOutputs
+IMAGE_PATH             = ../data/doxygenInputs \
```

Comment:
Interesting that the path change didn't affect anything on local build. Images show before and now.

Reply:
It was interesting that it built with the ../../ path when you consider the Doxyfile is in docs/doxygen and the images are in docs/data. It did not build correctly on my system, which is why I changed it.